modules/las/src/las-loader.ts

```diff
@@ -6,13 +6,31 @@ import type {Loader, LoaderOptions} from '@loaders.gl/loader-utils';
 const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
 
 export type LASLoaderOptions = LoaderOptions & {
-  las?: {
+  las: {
```

Comment:
`options.las` should be optional. You should not need to provide options. Look at other loaders for how this is handled by applying defaults.